### ISSUE TITLE
Feature Request: Keymap preview and new hotkeys

### ISSUE THREAD

**Issue by malammar** [beekeeper-studio/beekeeper-studio#214]
Love this app! Fills a massive need in the dev arsenal.

I would love to see expanded hotkey support and the ability to review available hotkeys.

- Tab forward/back: `Ctrl+PgUp` / `Ctrl+PgDown`
- Move focus between sidebar, results set: `Ctrl+B`
- Toggle focus from sidebar to active query editor: `Ctrl+B`
- Arrow key support in sidebar
- Disconnect: `Ctrl+Shift+W`

**Comment by rathboma** [comment]
Yes agreed! We need to do a big reworking of keyboard shortcuts, we don't even document them right now, not good.

I'll leave this here, but will eventually create a keyboard-shortcut meta-ticket to document all the keyboard shortcuts we want.

**Comment by goffyara** [comment]
+ ability to change `Run` hotkey

**Comment by malammar** [comment]
@rathboma I'm a Vue/Electron developer myself so I'd love to contribute

**Comment by rathboma** [comment]
Awesome! 

If you look at 'core-tabs.vue' it should be fairly easy to add the ctrl+pageup and ctrl+pagedown shortcuts. I think coreinterface.vue has some of the global keyboard shortcuts too

**Comment by lmeysel** [comment]
Would also love to have e.g. F5 for refresh view (or Ctrl+Enter would also be convinient, when a table is opened)

**Comment by lowercase00** [comment]
Cmd + R for refresh would also be great! I will take a look at 'core-tabs.vue', to see if I can contribute in any way.

**Comment by whatnotery** [comment]
@rathboma I would love a shortcut to toggle the sidebar. I did some research and it seems `ALT+S` would be open keyspace. Based on a quick search of the codebase and the electron documentation it seems it could be done by adding `accelerator: "Alt+S",` around [here](https://github.com/beekeeper-studio/beekeeper-studio/blob/36182f5e2d4c787a2d17868504e315685c779dec/apps/studio/src/common/menus/MenuItems.ts#L145) . If that's all it would take and you're cool with it I'd be happy to make a pull request (it would be my first contribution to an open source project :) )

**Comment by sunnz** [comment]
+1 for shortcut to disconnect
